gradio/examples.py

```diff
@@ -186,6 +189,7 @@ def __init__(
             samples=non_none_examples,
             type="index",
             label=label,
+            elem_id=elem_id,
```

Comment:
This change looks good to me but we need to rebase prior to merging in! Thanks @abidlabs !